dask_geomodeling/raster/spatial.py

```diff
@@ -132,7 +132,7 @@ class Dilate(BaseSingle):
 
     def __init__(self, store, values):
         values = np.asarray(values, dtype=store.dtype)
-        super(Dilate, self).__init__(store, values)
+        super().__init__(store, values.tolist())
```

Comment:
But why are they converted to an array in the first place?

Reply:
Note sure why, maybe to avoid issues with dtypes when comparing values in the `process` ?